includes/admin/css/geolocator.css

```diff
@@ -1,3 +1,90 @@
+/* Adjust spacing of geolocation metabox a bit */
+#wanderlist-geolocation {
+	margin-top: 15px;
+	margin-bottom: 0px;
+}
+
+#wanderlist-geolocation-map {
+	height: 300px;
+	position: relative;
+	width: 100%;
+}
+
+/* Style "find me" button */
+#wanderlist-locate-user {
+	background-color: #fff;
+	border: 1px solid rgba(0,0,0,.4);
+	border-radius: 3px;
+	box-shadow: none;
+	position: absolute;
+		top: 10px;
+		left: 220px;
+	z-index: 7;
+}
+
+#wanderlist-locate-user::before {
+	color: #404040;
+	content: '\f230';
```

Comment:
I don't think it's Unicode, but I've added a comment with a link to a Dashicons reference, in 0a00eec